docs/data/material/customization/how-to-customize/how-to-customize.md

```diff
@@ -135,26 +130,20 @@ You can rely on the following [global class names](/styles/advanced/#with-materi
 
 ## 2. Reusable style overrides
```

Comment:
Is it better to say `2. Reusable component`?

Reply:
Probably yes! Sounds better.